### PR TITLE
care cards center button, time and tags spacing

### PR DESCRIPTION
**Pull request by jen-p**

### CODE REVIEW

**Comment by craigmcginley** [comment]
Deploy preview for *loving-austin-5f2eff* ready!

Built with commit 7231519e5f81cf7e5ac4f86669c8b4aa294d0d7d

https://deploy-preview-129--loving-austin-5f2eff.netlify.com

File: src/case-studies/care-cards.md [hunk 1]
```diff
@@ -29,14 +29,13 @@ Care Cards put you in touch with habits to improve your health, life, and well-b
...
 * Nancy Shute, NPR.org: [If A Picture's Worth 1,000 Words, Could It Help You Floss?](http://www.npr.org/blogs/health/2014/03/28/295734262/if-a-pictures-worth-1-000-words-could-it-help-you-floss)
 * Patient Safety & Quality Healthcare: [Health Axioms: Turning Advice into Lifestyle](http://psqh.com/news-health-axioms-turning-advice-into-lifestyle) -
 
 
-<span class="text--uppercase text--gray text--bold text--spacing">Time:</span> 1 designer, part time, across 2 years
-<span class="text--uppercase text--gray text--bold text--spacing">Tags:</span> Patient Engagement, Open Source
-<a href="www.carecards.me" class="button button--primary button--lg margin-top--double margin-bottom--half">Get your own deck</a>
+<br/><span class="text--uppercase text--gray text--bold text--spacing">Time:</span> 1 designer, part time, across 2 years
+<br /><span class="text--uppercase text--gray text--bold text--spacing">Tags:</span> Patient Engagement, Open Source
+<div class="container container--justify-center"><a href="www.carecards.me" class="button button--primary button--lg margin-top--double margin-bottom--half ">Get your own deck</a></div>
```
Comment (craigmcginley) [inline src/case-studies/care-cards.md]: I think you need `http://` in the link here. Right now it's pointing to `<current-url>/www.carecards.me`

**Comment by craigmcginley** [comment]
@jen-p we might want to confirm links on other case studies don't suffer from the same issue here as well.

**Comment by jen-p** [comment]
> @jen-p we might want to confirm links on other case studies don't suffer from the same issue here as well.

fixed the link, will peruse through other case studies to double check their links

**Comment by jen-p** [comment]
other case studies look good, seems like this was the only one

**Pull request merged**